docs/source/howto/share_data.rst

```diff
@@ -0,0 +1,245 @@
+
+If the number of results to be exported is large, for example in a high-throughput study, use the ``QueryBuilder`` to add the corresponding nodes to a group ``my-results`` (see :ref:`how-to:data:organize:group`).
+Then export the group:
+
+.. code-block:: console
+
+    $ verdi export create my-calculations.aiida --groups my-results
+
+Publishing AiiDA archive files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+AiiDA archive files can be published on any research data repository, for example the `Materials Cloud Archive`_, `Zenodo`_, or the `Open Science Framework`_.
+When publishing AiiDA archive files on the `Materials Cloud Archive`_, you also get an interactive *EXPLORE* section, which allows peers to browse the AiiDA provenance graph directly in the browser.
+
+
+.. _Zenodo: https://zenodo.org
+.. _Open Science Framework: https://osf.io
+.. _Materials Cloud Archive: https://archive.materialscloud.org
+
+
+Importing an archive
+^^^^^^^^^^^^^^^^^^^^
+
+Use ``verdi import`` to import AiiDA archives into your current AiiDA profile.
+``verdi import`` accepts URLs, e.g.:
+
+.. code-block:: console
+
+    $ verdi import "https://archive.materialscloud.org/record/file?file_id=2a59c9e7-9752-47a8-8f0e-79bcdb06842c&filename=SSSP_1.1_PBE_efficiency.aiida&record_id=23"
+
+During import, AiiDA will avoid identifier collisions and node duplication based on UUIDs (and email comparisons for :py:class:~`aiida.orm.users.User` entries).
+By default, existing entities will be updated with the most recent changes.
+Node extras and comments have special modes for determining how to import them - for more details, see ``verdi import --help``.
+
+.. tip:: The AiiDA archive format has evolved over time, but you can still import archives created with previous AiiDA versions.
+    If an outdated archive version is detected during import, you will be prompted to confirm automatic migration of the archive file.
+
+    You can also use ``verdi export migrate`` to update existing archive files once and for all.
+
+.. tip:: In order to get a quick overview of an archive file *without* importing it into your AiiDA profile, use ``verdi export inspect``:
+
+    .. code-block:: console
+
+        $ verdi export inspect sssp-efficiency.aiida
+        --------------  -----
+        Version aiida   1.2.1
+        Version format  0.9
+        Computers       0
+        Groups          0
+        Links           0
+        Nodes           85
+        Users           1
+        --------------  -----
+
+    Note: For archive versions 0.2 and below, the overview may be inaccurate.
+
+
+.. _how-to:data:serve:
+
+Serving data through the REST API
+=================================
+
+The AiiDA REST API allows to query your AiiDA database over HTTP(S).
+
+.. note::
+
+    As of October 2020, the AiiDA REST API does *not* yet support workflow management.
+    This feature is, however, part of the `AiiDA roadmap <https://github.com/aiidateam/aiida-core/wiki/AiiDA-release-roadmap>`_.
+
```

Comment:
Please add a link here:
```
.. _how-to:data:serve:launch:
```
I'd like to reference it

Reply:
thanks, will do!

I just fixed the section name, it will be `how-to:share:serve:launch`